src/main/java/ru/skypro/homework/exceptions/AdNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package ru.skypro.homework.exceptions;
+
+public class AdNotFoundException extends Exception {
```

Comment:
```suggestion
public class AdNotFoundException extends RuntimeException {
```
исключения лучше делать непроверяемыми, чтобы не захламлять код try/catch/throws